modin/core/dataframe/algebra/default2pandas/rolling.py

```diff
@@ -37,9 +37,9 @@ def _build_rolling(cls, func):
             Function that takes pandas DataFrame and applies `func` on a rolling window.
         """
 
-        def fn(df, rolling_args, *args, **kwargs):
+        def fn(df, rolling_kwargs, *args, **kwargs):
```

Comment:
reworked `rolling_args` into `rolling_kwargs` everywhere for interoperability between `Rolling` and `RollingGroupby` constructors